src/detection/os/os_linux.c

```diff
@@ -157,6 +157,13 @@ static bool detectDebianDerived(FFOSResult* result)
         ffStrbufSetNS(&result->versionID, versionEnd - versionStart, result->prettyName.chars + versionStart);
         return true;
     }
+    else if (ffStrbufStartsWithS(&result->name, "CuerdOS GNU/Linux"))
+    {
+        ffStrbufSetS(&result->name, "CuerdOS GNU/Linux");
+        ffStrbufSetS(&result->id, "cuerdos");
+        ffStrbufSetS(&result->idLike, "debian");
```

Comment:
Doesn't your os-release file provide these info?

Reply:
Yes, this is how we have it set up

![imagen](https://github.com/fastfetch-cli/fastfetch/assets/134963245/efa893ea-457d-471a-a0c3-fd0ef6b4568c)

Reply:
So don't hard code these info. Remove these `ffStrbufSetS`s

Reply:
Did it